src/transformations.jl

```diff
@@ -26,17 +36,27 @@ transform(t::AbstractDataTransform, x::AbstractArray{<:Real,2}) = transform!(sim
 Perform an in-place reconstruction into an original data scale from a row-transformed
 vector or matrix `y` using `t` transformation.
 """
-reconstruct!(t::AbstractDataTransform, y::AbstractArray{<:Real,1}) = reconstruct!(y, t, y)
-reconstruct!(t::AbstractDataTransform, y::AbstractArray{<:Real,2}) = reconstruct!(y, t, y)
+function reconstruct!(t::AbstractDataTransform, y::AbstractVecOrMat{<:Real}; dims::Union{Integer,Nothing}=nothing)
```

Comment:
Reconstruction shouldn't have a `dims` argument. It makes no sense. Information about which dimension was used in the transformation should come from the `DataTransform` object.

Reply:
OK